src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs

```diff
@@ -162,25 +162,43 @@
         <Directory Id="INSTALLLOCATION" ShortName="ANCM" Name="$(var.ProductName)">
           <Directory Id="VersionDir" Name="$(var.ProductVersionString)">
             <Component Id="AspNetCoreModule" Guid="84ed6ce6-c8a3-4fa8-a872-c98a1d15dd4f" Win64="$(var.IsWin64)">
```

Comment:
Is there a reason why the component GUIDs aren't generated and why you have the same GUIDs for both 64-bit and 32-bit components?

Reply:
You can't install the 64 and 32 bit msis on the same machine, so we should be okay in that scenario.

Why GUIDs aren't generated is a different issue that should probably be resolved.